src/main/resources/db/migration/V6.4.0.1.sql

```diff
@@ -1,4 +1,4 @@
 -- Adding a case-insensitive index related to #11003
 --
 
-CREATE UNIQUE INDEX IF NOT EXISTS INDEX_DVOBJECT_authority_protocol_upper_identifier ON dvobject (authority, protocol, UPPER(identifier));
\ No newline at end of file
+CREATE UNIQUE INDEX IF NOT EXISTS INDEX_DVOBJECT_authority_protocol_upper_identifier ON dvobject (authority, protocol, UPPER(identifier));
```

Comment:
Will removing the newline make Flyway calculate a different checksum and refuse to deploy this? Should we `git checkout` (revert) the file back to how it was?

Reply:
Yes, good call! 
It did fail to deploy for me this morning, and I just deleted the flyway history entry and deployed again without thinking about it much. I must have introduced this when resolving a conflict with the cherry-picked commit from my other branch.

Reply:
Interestingly, it actually looks like Flyway is smart enough to see that it's still the same script, despite the extra newline - according to a quick experiment. It must be stripping newlines (and maybe extra whitespaces and comments?) before taking the checksum - ? 
my failure to deploy this morning appeared to be related to something else.